packages/bruno-app/src/utils/codegenerator/har.js

```diff
@@ -38,7 +38,7 @@ const createPostData = (body) => {
       mimeType: contentType,
       params: body[body.mode]
         .filter((param) => param.enabled)
-        .map((param) => ({ name: param.name, value: param.value }))
+        .map((param) => ({ name: param.name, value: encodeURIComponent(param.value) }))
```

Comment:
Theoretically param name can also contains spaces or some other characters which should be encoded (at least I'm not aware of such RFC which does not allow it). So don't we want to apply the same also for param name ?

Reply:
We certainly could. I've never come across such case, but escaping the name should do no harm anyways.

Reply:
@martinsefcik as a second thought, it makes more sense to just encode the whole URL. That would catch any corner cases. Does that make sense for you?